cmake/iotjs.cmake

```diff
@@ -566,12 +566,17 @@ if("${BIN_INSTALL_DIR}" STREQUAL "")
   set(BIN_INSTALL_DIR "bin")
 endif()
 
+if("${INC_INSTALL_DIR}" STREQUAL "")
+  set(INC_INSTALL_DIR "include/iotjs")
+endif()
+
 # Configure the iotjs executable
 if(NOT BUILD_LIB_ONLY)
   set(TARGET_IOTJS iotjs)
   message(STATUS "CMAKE_BINARY_DIR        ${CMAKE_BINARY_DIR}")
-  message(STATUS "BINARY_INSTALL_DIR      ${INSTALL_PREFIX}/bin")
-  message(STATUS "LIBRARY_INSTALL_DIR     ${INSTALL_PREFIX}/lib")
+  message(STATUS "BINARY_INSTALL_DIR      ${INSTALL_PREFIX}/${BIN_INSTALL_DIR}")
+  message(STATUS "LIBRARY_INSTALL_DIR     ${INSTALL_PREFIX}/${LIB_INSTALL_DIR}")
+  message(STATUS "INCLUDE_INSTALL_DIR     ${INSTALL_PREFIX}/${INC_INSTALL_DIR}")
```

Comment:
Shouldn't we replace the `INSTALL_PREFIX` with  `CMAKE_INSTALL_PREFIX` ?

Reply:
well it was introduced in 99ec37101 may this be changed in other change ?

My install files relies on debhelper DESTDIR there is no need tweak:

https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html

Reply:
So the main thing in this PR would be the new `INC_INSTALL_DIR` variable? If so I'm fine with this change. We'll adapt the INSTALL_PREFIX later on.

Reply:
> So the main thing in this PR would be the new `INC_INSTALL_DIR` variable? If so I'm fine with this change.

Thanks the problem is that it's blocked by @akosthekiss change request which can be only acknowledged by himself but I am not able to reach him, can anyone help ?

Reply:
I'm not it.